src/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,69 @@
+
+The Proof-of-Authority simply requires to trust a subset of authorized signers that are set in the [genesis block](/glossary/#genesis-block), they have the same power in all situations as signers, no signer has exclusive privileges than others in the blockchain as you might think(unless new implementation follow this concept is introduced), the idea behind reputation staking is every authorized signer is well-known to everyone through things like KYC, or by having a well known org shouting they will be the only signer in their blockchain, this way if someone did something wrong, it can be detected who and what their identity is, i.e there will be no attacks on the consensus mechanism by anonymous signers.
+
+The Proof-of-Authority has different implementations, but the ethereum standard implementation is clique that implements [eip-225](https://eips.ethereum.org/EIPS/eip-225), it has the most developer-friendly and easy to implement standard that all clients syncing types are supported, unlike a previously used implementation (Aura) that was implemented in openethereum, which fast,light syncing are not supported because smart contracts are used to store the authorized signers, clique instead only reuse obsoleted headers to solve the issues related to voting to add and remove signers.
+
+## how it works {#how-it-works}
+
+clique standard simply make a use of the obsolete headers: extradata that was used only for fun, miner became obsolete as there are no miners in POA, and nonce became also obsolete because the authorities the order of users transactions, extradata is used to store the authorized signers set, miner and nonce are used to vote for adding or removing signers from the network, where every epoch transition flushes all pending votes. The block proposal is selected sequentially for each block, but in fact any other authorized signer can propose block even if it's not his turn, but they have limit of signing only one block between K blocks when N are the authorized signers.
+
+## PROOF-OF-AUTHORITY AND SECURITY {#poa-security}
+
+### Attack vector: Malicious signer
+
+It may happen that a malicious user gets added to the list of signers, or that a signer key/machine is compromised. In such a scenario the protocol needs to be able to defend itself against reorganizations and spamming. The proposed solution is that given a list of N authorized signers, any signer may only mint 1 block out of every K. This ensures that damage is limited, and the remainder of the miners can vote out the malicious user.
```

Comment:
```suggestion
It may happen that a malicious user gets added to the list of signers, or that a signing key/machine is compromised. In such a scenario the protocol needs to be able to defend itself against reorganizations and spamming. The proposed solution is that given a list of N authorized signers, any signer may only mint 1 block out of every K. This ensures that damage is limited, and the remainder of the miners can vote out the malicious user.
```